content/en/docs/distributions/ibm/pipelines.md

```diff
@@ -38,15 +38,18 @@ def echo_pipeline(
 * You will be using the Kubeflow Pipelines with Tekton SDK ([`kfp-tekton`](https://pypi.org/project/kfp-tekton/)) v0.4.0 or above.
 * If you have deployed Kubeflow on IBM Cloud using the 
 [`kfctl_ibm.v1.2.0.yaml`](https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm.v1.2.0.yaml)
-manifest you can configure ([`kfp-tekton`](https://pypi.org/project/kfp-tekton/)) SDK for a single user as follows:
+manifest you can configure ([`kfp-tekton`](https://pypi.org/project/kfp-tekton/)) SDK to list all your Kubeflow Pipelines experiments as follows:
 
 ```python
 from kfp_tekton import TektonClient
 
 KUBEFLOW_PUBLIC_ENDPOINT_URL = 'http://<Kubeflow_public_endpoint_URL>'
```

Comment:
Suggestion:

Maybe we should rename `<Kubeflow_public_endpoint_URL>` to `<YOUR_KF_PUBLIC_ENDPOINT_URL>`. Otherwise, it's almost exactly like `KUBEFLOW_PUBLIC_ENDPOINT_URL`, which may confuse some users.

```suggestion
KUBEFLOW_PUBLIC_ENDPOINT_URL = 'http://<YOUR_KF_PUBLIC_ENDPOINT_URL>'
```

Reply:
Thanks for the review and suggestion. Updated so hopefully it is clearer now.